app/travel_processor/package.json

```diff
@@ -25,6 +28,13 @@
     "karma-ui5": "^4.0.1",
     "puppeteer": "^23.0.2",
     "ui5-middleware-simpleproxy": "^3.2.16",
-    "ui5-task-zipper": "^3.1.4"
+    "ui5-task-zipper": "^3.1.4",
+    "@sap/ux-ui5-tooling": "1",
+    "@sapui5/types": "~1.130.0",
+    "ui5-tooling-transpile": "^3.3.7",
+    "typescript": "^5.1.6",
+    "@typescript-eslint/eslint-plugin": "^7.1.1",
+    "@typescript-eslint/parser": "^7.1.1",
+    "@sap-ux/eslint-plugin-fiori-tools": "^0.4.0"
```

Comment:
```suggestion
    "@ui5/cli": "^3.0.0",
    "@sap/ux-ui5-tooling": "1",
    "@sapui5/types": "~1.130.0",
    "ui5-tooling-transpile": "^3.3.7",
    "typescript": "^5.1.6",
    "@typescript-eslint/eslint-plugin": "^7.1.1",
    "@typescript-eslint/parser": "^7.1.1",
    "@sap-ux/eslint-plugin-fiori-tools": "^0.4.0"
```

Reply:
Keeping task zipper and ui5 cli 4 as discussed